bears/generate_package.py

```diff
@@ -113,7 +117,18 @@ def main():
     else:
         bear_version = repr(bear_version) + '.' + str(int(time.time()))
 
-    for bear_file_name in sorted(set(glob('bears/**/*Bear.py'))):
+    local_files = sorted(set(glob('bears/**/*Bear.py')))
+
+    if args.bear:  # Checks if bear name entered is available on the machine
+        for f in local_files:
```

Comment:
Lets say there are 2 bears coalaBear and coalaliintBear, giving input as 'coala' would lead to both bears to be uploaded.